.github/ISSUE_TEMPLATE/simeng-bug-or-issue-report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: SimEng Bug or Issue Report
+about: Report a bug or un-expected SimEng behaviour
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Check List**
+ [ ] The binary I am trying to run has been compiled statically for either RV64 or AArch64.
```

Comment:
Doing a draft issue template, and assuming you want these to be markdown checkboxes, it looks like these should be `- [ ]` to render properly